superset/charts/data/api.py

```diff
@@ -130,6 +135,7 @@ def get_data(self, pk: int) -> Response:
             "format", ChartDataResultFormat.JSON
         )
         json_body["result_type"] = request.args.get("type", ChartDataResultType.FULL)
+        json_body["force"] = request.args.get("force", False)
```

Comment:
In order to handle `false` as a string in the force, an explicit type conversion should be in here. for example:
```type
        json_body["force"] = explict_bool_conversion(request.args.get("force"))
```

Reply:
umm I think that is taken care by `_create_query_context_from_form` function below using mashmallow schema's load function.
I tested with false too and it's working as expected, do you still need this conversion here?

Reply:
if exists another deserialization helper on the json_body, the default value of `request.args.get("force")` should be None instead of `request.args.get("force", False)`, because the `request.args.get("force")` will always return `Optional[str]`.

Reply:
ok updated.

Reply:
if we set force to be `None` then masrhmallow raises error saying it should be a boolean.

Reply:
add `allow_none` argument in the rule. 
```
    force = fields.Boolean(
        description="Should the queries be forced to load from the source. "
        "Default: `false`",
        allow_none=True
    )
```